### ISSUE TITLE
Remove webhook validation on v2 api

### ISSUE THREAD

**Issue by fzf** [plaid/link#278]
TLDR; We would like Plaid to suppress the validation for the webhooks parameter on the server either for our account or for everyone. The upgrade to v2 adds this validation but we have clients in the wild that are passing `webhook=undefined` and no `apiVersion` when the v1 api is deprecated the server defaults to `apiVersion=v2` because we do not set it. Then we get this very generic error exposed to our users instead of a configuration error.

*Please do not include your API `secret` or `public_key` in any issue.*
---
### A detailed description of the steps to reproduce the issue

* Visit `https://cdn.plaid.com/link/v2/stable/link.html?key=PUBLIC_KEY&env=production&product=auth&clientName=ClientName&isWebView=true&isMobile=true&webhook=undefined&selectAccount=true&apiVersion=v2`
* Enter any credentials
* Get a generic error message shown to the user and a invalid webhook error on the console

### What you expected to see?
Plaid link widget works but just does not set webhook

### What you saw instead?
Generic error message

<details><summary>Screenshots that show the state of the UI (if applicable)</summary>
![image 2](https://user-images.githubusercontent.com/1462357/46380787-88d6cb80-c658-11e8-889d-b9c6aa879f8a.png)
</details>

---

When submitting an issue for Link on the web or within a Webview, please include the following information:

|       Name               | Value                                             |
|-------------------|--------------------------------|
| Link `env`             |production                                     |
| Browser                 |Chrome but not browser specific|
| Operating system |OSx but not OS specific               |

#### List of used 3rd party libraries (e.g. CocoaPods):

* [React Native Plaid](https://github.com/catalinmiron/react-native-plaid-link) v1.3.3

**Comment by edwinychu** [comment]
@pbernasconi @afh This was the issue Viktoriya and I flagged yesterday when we synced. Seems like the context Fletcher has provided is sufficient, but let me know if you need any more.

**Comment by pbernasconi** [comment]
Hi @fzf please excuse the delayed response – just wanted to acknowledge that we are looking into possible solutions for your issues transitioning to the new API.

We have a few options we're considering, and I'm confident we'll be able to push a fix to Link Web next week. Thanks for your patience and sorry for your troubles as your migrate!

**Comment by fzf** [comment]
@pbernasconi this seems to have broken the integration, now we just see the loading screen
